pyproject.toml

```diff
@@ -64,3 +64,6 @@ module = [
     "setproctitle",
 ]
 ignore_missing_imports = true
+
+[tool.pytest.ini_options]
+asyncio_mode = "auto"
```

Comment:
I'm surprised this worked! https://github.com/pantsbuild/pants/pull/12679

That's great news, I can revive that PR now

Reply:
That was likely fixed by #14784